docs/features/publishing-events.md

```diff
@@ -39,6 +39,22 @@ Alternatively you can publish a list of events by using
 await eventGridPublisher.PublishMany(events);
 ```
 
+### Publishing Raw Events
+
+We provide the capability to push events without a schema based on a raw JSON string.
+
+```csharp
+// Created via EventGridPublisherBuilder.
+EventGridPublisher eventGridPublisher = ...
+
+string licensePlate = "1-TOM-337";
+string eventSubject = $"/cars/{licensePlate}";
+string eventId = Guid.NewGuid().ToString();
+string rawEventPayload = $"{{ \"licensePlate\": \"{licensePlate}\"}}";
```

Comment:
Shouldn't this be with just one { }?
```suggestion
string rawEventPayload = $"{ \"licensePlate\": \"{licensePlate}\"}";
```

Reply:
No, since the `$` will otherwise see this as a way to insert a string.
Like `$ blabla {stringVariable} blabla"

Reply:
Ha, good point! Unfortunately, our documentation engine does not like it.

Maybe use good ol' String.Format instead?

YOu can run the docs locally as well if you want https://jekyllrb.com/docs/